feat(changelog): billing-added-api-billing-v2-beta 2023-12-20 (#2551)
scaleway/docs-content: changelog/december2023/2023-12-20-billing-added-api-billing-v2-beta.mdx

---

> title: API BILLING V2 BETA

Reviewer: ```suggestion
title: The Billing API V2 Beta is now available!
```

status: added
author:

>     fullname: 'Join the #billing-cost-management on scaleway slack community'

Reviewer: ```suggestion
    fullname: 'Join the #billing-cost-management on Scaleway Slack community'
```

    url: 'https://slack.scaleway.com'
date: 2023-12-20
category: billing
product: billing
---

> Hello folks, 

Reviewer: ```suggestion
```

> We are pleased to introduce the new beta version of Billing API v2 beta 1

Reviewer: ```suggestion
We are pleased to introduce the new beta version of the Billing API v2.
```

Scaleway's Billing API gives you an overview of your cloud consumption and allows you to monitor expenses.

What's New?

> - Monthly Consumption History: Now, access the complete history of your monthly consumption. Until now, only data from the current month was available.

Reviewer: ```suggestion
- Monthly Consumption History -  access the complete history of your monthly consumption. Until now, only data from the current month was available.
```

> - Granularity of Costs by Project, Product Category, and Month: We've introduced increased granularity, allowing you to view costs based on the project, product category, and month.

Reviewer: ```suggestion
- Granularity of Costs by Project, Product Category, and Month - we have introduced increased granularity, allowing you to view costs based on the project, product category, and month.
```

> [Retrieve our documentation here](https://www.scaleway.com/en/developers/api/billing/) 

Reviewer: ```suggestion
Refer to the [Billing API documentation](https://www.scaleway.com/en/developers/api/billing/) for more information.
```
